internal/host/fioperfcheckcmd.go

```diff
@@ -16,19 +17,26 @@ const (
 	FioDurationThreshold    int64 = 1000
 )
 
+type FioPerfCheckConfig struct {
+	ServiceBaseURL       string
+	ClusterID            string
+	HostID               string
+	UseCustomCACert      bool
+	FioPerfCheckImage    string
+	SkipCertVerification bool
+	Path                 string
+	DurationThreshold    int64
```

Comment:
Please add units to the variable name (also on the API where the agent returns the value to the service)

Reply:
Done